src/System.Security.Cryptography.Primitives/src/Resources/Strings.resx

```diff
@@ -106,6 +106,9 @@
   <data name="Cryptography_InvalidPaddingMode" xml:space="preserve">
     <value>Specified padding mode is not valid for this algorithm.</value>
   </data>
+  <data name="Cryptography_MissingIV" xml:space="preserve">
+    <value>The cipher mode specified requires that an initialization vector (IV) be used.</value>
```

Comment:
The resource strings that are being added suggest a bad "common factoring".

Reply:
Well, I can skip that one method. Will update the PR.